MapboxNavigation/RouteMapViewController.swift

```diff
@@ -347,6 +343,9 @@ class RouteMapViewController: UIViewController {
     @objc func updateInstructionsBanner(notification: NSNotification) {
         guard let routeProgress = notification.userInfo?[RouteControllerNotificationUserInfoKey.routeProgressKey] as? RouteProgress else { return }
         instructionsBannerView.updateInstruction(routeProgress.currentLegProgress.currentStepProgress.currentVisualInstruction)
+        lanesView.update(for: routeProgress.currentLegProgress.currentStepProgress)
+        nextBannerView.update(for: routeProgress.currentLegProgress.currentStepProgress)
```

Comment:
Let's make all instruction view have the same signature: `foo.updateInstruction(for:)`.

Reply:
@bsudekum that's not applicable in that context. If the function was using a route step progress, that would make sense. However, this function is currently updating the instruction with `currentVisualInstruction`:

https://github.com/mapbox/mapbox-navigation-ios/pull/1514/files#diff-7bae0cf60f54997d0d5e86fe02b5cddbL96

It will look weird to compose the function like this:
`public func updateInstruction(for instruction: VisualInstructionBanner?)`

Yeah, completely out of place and definitely not well composed.